src/main/java/com/bunq/sdk/exception/ApiException.java

```diff
@@ -1,37 +1,9 @@
 package com.bunq.sdk.exception;
 
-import java.util.List;
 
-/**
- * Exception triggered by API requests failed on the server side.
- */
-public class ApiException extends RuntimeException {
+public class ApiException extends BunqError {
```

Comment:
As with Python, let's make `ApiException` the base of it all. We do not need the `BunqError` :).